spec/views/components/header_notice.html.erb_spec.rb

```diff
@@ -1,146 +0,0 @@
-RSpec.describe "header-notice component" do
```

Comment:
What was the reason for not having a `_nhs_notice.html.erb_spec`?

Reply:
It didn't strike me as something that would be conventional to test. All of the code paths are executed implicitly as part of other tests.

Reply:
I am surprised... I thought you were a view test fan @kevindew! It did seem very rigidly tested before which was maybe overkill but then testing rendering the data from the `coronavirus_landing_page` feels maybe too vague?

Reply:
Haha, I wouldn't really describe myself as a fan of them - but it is preferable, if they contain logic, to test them at a unit level view test rather than using a higher level test to unit test them.

I think essentially all you really want testing wise from views is that you want all code paths exercised (which is often all that is sufficient for most unit tests) which just gives you the basic reassurance that all the code executes and there's not some hidden typo or dependency that isn't exercised until runtime. 

There seems low value in testing that a view outputs particular HTML as it just basically confirms you typed what you did in the code.